services/madoc-ts/src/frontend/site/pages/tasks/review-listing/review-listing-page.tsx

```diff
@@ -312,6 +314,7 @@ function SingleReviewTableRow({ task, active, page }: { task: CrowdsourcingTask;
 
   return (
     <ThickTableRow
+      tabIndex={0}
```

Comment:
There's a [hook in Manifest Editor](https://github.com/digirati-co-uk/iiif-manifest-editor/blob/583b28cd14aedadf08776e5c677b9a344f4b95ab/manifest-editor/src/hooks/use-keyboard-list-navigation.ts) that we could use here. 

It works like this:
```jsx
const containerProps = useKeyboardNavigation('data-something-unique');

return (
  <div {...containerProps}>
    <div tabIndex={0} data-something-unique="0">First</div>
    <div tabIndex={0} data-something-unique="1">Second</div>
    <div tabIndex={0} data-something-unique="2">Third</div>
  </div>
);
```

and will allow you to use the arrow keys to navigate the list.